docs/snippets/providers/netdata-snippet-autogenerated.mdx

```diff
@@ -7,3 +7,23 @@ Do not edit it manually, as it will be overwritten */}
 This provider can't be used as a "step" or "action" in workflows. If you want to use it, please let us know by creating an issue in the [GitHub repository](https://github.com/keephq/keep/issues).
 
 
+
+## Connecting via Webhook (omnidirectional)
+This provider supports webhooks.
+
+
+To send alerts from Netdata to Keep, Use the following webhook url to configure Netdata send alerts to Keep:
+
+1. In Netdata, go to Space settings.
+2. Go to "Alerts & Notifications".
+3. Click on "Add configuration".
+4. Add "Webhook" as the notification method.
+5. Add a name to the configuration.
+6. Select Room(s) to apply the configuration.
+7. Select Notification(s) to apply the configuration.
+8. In the "Webhook URL" field, add KEEP_BACKEND_URL/alerts/event/PROVIDER_TYPE.
```

Comment:
```suggestion
8. In the "Webhook URL" field, add KEEP_BACKEND_URL/alerts/event/netdata
```

this should apply to all

Reply:
You're changing auto-generated files. I will update the generator.